CSharpMath.Forms/Buttons.cs

```diff
@@ -7,27 +7,56 @@ public abstract class BaseButton<TView, TPainter, TContent> : ImageButton
     where TView : BaseView<TPainter, TContent>
     where TPainter : Rendering.FrontEnd.Painter<SKCanvas, TContent, SKColor>, new()
     where TContent : class {
+    private Color _textColor = Color.Black;
+    public Color TextColor {
+      get => _textColor;
+      set {
+        _textColor = value;
+        SetImageSource();
+      }
+    }
     public BaseButton() {
       Aspect = Aspect.AspectFit;
-      // Color.Default will be ugly: https://github.com/verybadcat/CSharpMath/issues/111
       BackgroundColor = Color.Transparent;
-      Source = ImageSource.FromStream(() => {
-        if (Content is { } c) {
-          var latex = c.Painter.LaTeX;
-          if (c.Painter.FontSize is Rendering.FrontEnd.PainterConstants.DefaultFontSize)
-            // Have a clear output by default
-            c.Painter.FontSize = Rendering.FrontEnd.PainterConstants.LargerFontSize;
-          // Appropriate positioning for non-full characters, e.g. prime, degree
-          // Also acts as spacing between MathButtons next to each other
-          // TODO: Implement and use \phantom
-          c.Painter.LaTeX = @"{\color{#00000000}|}" + latex + @"{\color{#00000000}|}";
-          var stream = c.Painter.DrawAsStream();
-          c.Painter.LaTeX = latex;
-          return stream;
-        }
-        return null;
-      });
+      SetImageSource();
+    }
+
+    private void SetImageSource() => Source = ImageSource.FromStream(() => {
+      if (Content is { } c) {
+        var painter = c.Painter;
+        var originalLatexString = painter.LaTeX;
+        HaveLargerFontSizeByDefault(painter);
+        SetTextColorIfRequested(painter);
+        FixPositionOfNonFullCharacters(painter);
+        var stream = painter.DrawAsStream();
+        painter.LaTeX = originalLatexString;
+        return stream;
+      }
+      return null;
+    });
```

Comment:
This function can be inlined into the property setter as well by initializing `TextColor` in the constructor.